### DISCUSSION TITLE
Sesiones caducadas

### DISCUSSION THREAD

**Discussion by Cholo68** [Paco8/plugin.video.movistarplus#75] [General]
Buenos dias, ante todo darte la enhorabuena y las gracias por el gran addon que nos ofreces.
Tengo una pregunta, he comprobado que con éste addon si estás aprox 7 dias si entrar caduca la sesion (creo que con el de Orange no pasa o por lo menos tan a menudo) y hay que volver a meter los credenciales de nuevo, si entras mas habitualmente eso no ocurre. Hay alguna otra forma  para que no caduque sesion y haya que volver a meter los datos? Muchas gracias

**Comment by Paco8** [comment]
Es cosa de movistar, pasa lo mismo en su web. El token de acceso caduca a los 10 días. Si entras antes en el addon el token se renueva por otros 10 días pero si estás 10 días sin entrar tendrás que volver a meter las credenciales.

**Reply by Tarraco1** [reply]
Buenas, hoy e querido entrar a movistar y no puedo me dice que no se puede acceder sabes a que puede ser no hay forma de entrar a la cuenta te da error..!!

**Reply by Tarraco1** [reply]
Feliz Navidad

**Reply by gomdesk** [reply]
Buenas tardes,lo primero agradecer el trabajo que supone crear y mantener este addon que nos facilita el poder verlo en otro dispositivo extra.
A mi tb me da error de "la sesion a caducado"... no se a que se debera exactamente.

Felices Fiestas a todos y os deseo lo mejor para vosotros y para vuestros seres queridos!!!

**Reply by Paco8** [reply]
[No hace logging](https://github.com/Paco8/plugin.video.movistarplus/issues/78)

**Comment by Cholo68** [comment]
Muchas gracias Paco, sabia que era alrdedor de ese numero de dias pero no sabia que eran exactamente 10 dias, lo tendre en cuenta.
En Orange tiene caducidad el tokken?

**Reply by Paco8** [reply]
En Orange el token dura dos o tres días pero el addon hace login automático (su web también).

**Comment by Cholo68** [comment]
Ah pues genial entonces, ya podia hacer movistar lo mismo jeje. Lo que me ocurrio el otro dia con Orange fue un error que no se a que fue debido, lo pongo en el hilo de Orange. Muchas gracias por tu respuesta
